plugins/backup/backup-manager

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# Example config:
+#	[backup-manager]
+#	user root
+#	env.backup_dir /path/to/your/backups/
+#	env.lifetime 2
+#	env.archive_pattern *.tar.bz2
+#	env.backup_number 4
+
+# Configuration directives, edit before first use.
+BACKUP_DIR=${backup_dir:-/data/backup}
+ARCHIVE_PATTERN="${archive_pattern:-*.tar.bz2}"
+# How old backups should be considered as non-yound anymore in [days].
+LIFETIME=${lifetime:-2}
+# Critical states will be issued when the number of fresh backups archives is below `backup_number`,
+# and warnings below `backup_number*lifetime`
+CRIT=${backup_number:-1}
+WARN=$((${CRIT}*${LIFETIME}))
```

Comment:
You can remove the $ and the curly braces from arithmetic expressions. This should increase readability.